gunicorn/glogging.py

```diff
@@ -199,7 +199,9 @@ def setup(self, cfg):
                 self.access_log, cfg, self.syslog_fmt, "access"
             )
 
-        if cfg.logconfig:
+        if cfg.logconfigdict:
+            dictConfig(cfg.logconfigdict)
```

Comment:
Yep, using dictConfig for the default case seems sensible, at least if fileConfig will be eventually removed. I'll take a closer look at this.